src/main/java/com/superzanti/serversync/GUIJavaFX/PaneOptions.java

```diff
@@ -2,7 +2,7 @@
 
 import com.superzanti.serversync.ServerSync;
 import com.superzanti.serversync.config.SyncConfig;
-import com.superzanti.serversync.util.Logger;
+import com.superzanti.serversync.util.ServerSyncLogger;
```

Comment:
Considering that nothing about the callsite of this class appears to have changed it would have been much easier to review the changeset if the name of the class remained unchanged or was changed as part of a different commit.